aiohttp_admin/security.py

```diff
@@ -71,16 +71,42 @@ def __init__(self, username, password, permissions=None):
 
 class DummyTokenIdentityPolicy(AbstractIdentityPolicy):
 
+    def __init__(self, server_secret=None):
+        '''
+            Makes identity tokens using HMAC(SHA-512) over a
+            server-side secret.
+
+            Provide a secret (20+ bytes) or we'll pick one
+            at runtime.
+        '''
+        from hmac import HMAC
```

Comment:
may be better to move implementation to separate contrib module? and add hmac to `extras_require` in setup.py